src/pages/MatchCreation/MatchCreation.tsx

```diff
@@ -58,9 +57,8 @@ export const MatchCreation = () => {
     }
 
     mutate(match)
-
-    if (isSuccess) {
-      navigate(EDIT_MATCH.replace(':id', data.id.toString()))
+    if (!isError) {
```

Comment:
I commented this line on your previous MR. Please be mindful of those points here too